### ISSUE TITLE
Fortpy fpy_read_integer_2d finds to many values in the arrays.

### ISSUE THREAD

**Issue by wsmorgan** [rosenbrockc/fortpy#40]
When reading in a 2d array fortpy uses call fpy_linevalue_count(filename, commentchar, nlines, nvalues) to find the number of lines and values. At present it finds the number of lines correctly but fails on the number of value. For example if this file is passed in:
0	
0	
0	
it finds these as the lines and values respectively:
3
2
or alternatively if this file is read in:
0	1	2	3	4	5	6	7	8	9	
0	1	2	3	4	5	6	7	8	9	
0	1	2	3	4	5	6	7	8	9	
0	1	2	3	4	5	6	7	8	9	
0	1	2	3	4	5	6	7	8	9	
it finds the lines and values to be:
5
20

so far it seems like it counts each value twice. It then allocates to much space for the array triggering an EOF error.

**Comment by rosenbrockc** [comment]
Excellent, thanks Wiley. This is probably what was causing that mysterious BCS problem we had last year in the stand-alone solver. My guess is that it has something to do with recognizing different whitespace ASCII (or non-ASCII) characters. I'll look into it.

**Comment by rosenbrockc** [comment]
Wiley, I tried to reproduce the behavior you reported and I am not getting the same answers. Maybe we can look at this together tomorrow. Here is my driver:

```fortran
program fcheck
  use fortpy
  integer :: nlines, nvalues

  call fpy_linevalue_count('data.in', '#', nlines, nvalues)
  print *, nlines, nvalues  
end program fcheck
```

which produces `           5          10`.

**Comment by rosenbrockc** [comment]
There was indeed a problem but it is addressed in Revision 1.6.0.
